### ISSUE TITLE
Isaac Review Ticket Crossover

### ISSUE THREAD

**Issue by Isaac-Le** [tonyhieu/csp-anthonys-harem#33]

**Comment by Chom642** [comment]
Crossover: Issac Le
[Create Task](https://github.com/danaylevy2004/thesuperidolenthusiasts/wiki/Chris's-create-task-collaborated-with-Matthew)
[Create task video](https://user-images.githubusercontent.com/89278032/155925591-2eceafd0-1123-4080-a850-53c950c9b15c.mp4)
[Team video](https://www.youtube.com/watch?v=W2BwHXSeh-I)
[Commits](https://github.com/danaylevy2004/thesuperidolenthusiasts/commit/fa590c985a6d8bdcef0706894c9ee17c34671aff#diff-0a468f0a0199783ae45f876d11ef63bb8c2e7ca22a406e6d128a81bb65cc2ba9)
[Scrumboard](https://github.com/danaylevy2004/thesuperidolenthusiasts/projects/1)
Crossover Score:
9.1/10
- [x] Wiki/Jekyll Page with Design Ideas and Usage - 1/1
- [x] Create Task PBL Feature - 1/1
- [x] Final Committed Program Code with Algorithms - 1/1
- [x] Written Responses - 1/1
- [x] 1 min video that meets CB create task requirements - 0.7/1

**Team Grading Considerations**
- [x] PBL theme consistent - 0.5/0.5
- [x] Sassy/Bootstrap - 0.4/0.5
- [x] Blueprint illustration and demonstration - 0.5/0.5
- [x] Crud Async Operation - 0.5/0.5
- [x] Indi PBL MVC features - 0.5/0.5

**Runtime features shown in 1 min video**
- [x] Highlight of best features - 0.5/0.5
- [ ] CRUD - 0.2/0.5
- [x] MVC - 0.5/0.5
- [x] Google Search or Google Translate - 0.5/0.5
- [x] Scrum Board Overview - 0.3/0.5

**Comment by Chom642** [comment]
Overall did a very good job. Create task video demonstrated runtime within a minute, and displayed all features. Google search on website works, however translation feature was missing. MVC was present but was unorganized and messy. Blueprints were also disorganized and hard to follow. Otherwise everything else looked fine.
